sklearn/pipeline.py

```diff
@@ -493,12 +679,14 @@ def fit_transform(self, X, y=None, **fit_params):
             sum of n_components (output dimension) over transformers.
         """
         result = Parallel(n_jobs=self.n_jobs)(
-            delayed(_fit_transform_one)(trans, name, X, y,
-                                        self.transformer_weights, **fit_params)
-            for name, trans in self.transformer_list)
+            delayed(_fit_transform_one)(trans, name, weight, X, y,
+                                        **fit_params)
+            for name, trans, weight in self._iter())
 
         Xs, transformers = zip(*result)
         self._update_transformer_list(transformers)
+        if not Xs:
```

Comment:
So what case is this? Sorry, I'm slow. If there are no transformers, I'd expect line 715 to crash.